app/controllers/application_controller.rb

```diff
@@ -367,6 +367,7 @@ def report_data
     if options.nil? || options[:view].nil?
       model_view = process_params_model_view(params, options)
       @edit = session[:edit]
+      options_from_session(options)
```

Comment:
> You see some where clauses depends on selected tree node, some on model and some are stored in session based on some deep logic and I was unable to locate them, hence I did not removed get_view method.

@karelhala I understand that the `get_view` logic is buried deeply in the views and hard to just extract it out. A heads-up about this issue earlier on would have been nice though.